src/test/scala/com/comcast/xfinity/sirius/uberstore/CompactionManagerTest.scala

```diff
@@ -125,6 +125,6 @@ class CompactionManagerTest extends NiceTest with BeforeAndAfterAll with TimedTe
   }
 
   override def afterAll() {
-    actorSystem.shutdown()
+    actorSystem.terminate()
```

Comment:
Minor concern on all of these tests is that since this is future and you are not waiting, are there any consequences to this still being shutdown while another test starts up?  Not sure, just curious if we should do the Await.ready for all tests.

Reply:
Agreed, that seems like a good idea.  I was keeping the code functionally identical to what was already there, and `actorSystem.shutdown()` also didn't wait, it just called `actorSystem.terminate()` and discards the `Future[T]`.